ui-tests/src/test/java/org/openmrs/reference/MergeVisitsTest.java

```diff
@@ -27,16 +26,17 @@
 public class MergeVisitsTest extends LocationSensitiveApplicationTestBase {
 
     private TestData.PatientInfo patient;
+  
 
     @Before
     public void setUp() {
         patient = createTestPatient();
-        new TestData.TestVisit(patient.uuid, TestData.getAVisitType(), getLocationUuid(homePage)).create();
-        new TestData.TestVisit(patient.uuid, TestData.getAVisitType(), getLocationUuid(homePage)).create();
+        new TestData.TestVisit(patient.uuid,"Facility Visit", getLocationUuid(homePage)).create();
```

Comment:
any reason for this change @HerbertYiga ?

Reply:
The major reason of changing this is that with the orignal changes,the data set is created once,so we needed a way of making sure that we create it twice ie so that we have the two visists created